tests.py

```diff
@@ -39,17 +39,10 @@
     from matplotlib import test
 
     parser = argparse.ArgumentParser(add_help=False)
-    parser.add_argument('--no-network', action='store_true',
```

Comment:
Might be better to leave this for the sake of anyone who needs to suppress network access and test more than one version of matplotlib?

Reply:
They can always use `-m 'not network'`; mentioned this in changelog.